src/node_lib.h

```diff
@@ -26,17 +32,23 @@ namespace node::lib {
 
     /*
     Starts the Node.js engine without a concrete script file to execute.
-    *Important*: This requires the C++ developer to call `process_events()` periodically.
+    *Important*: This requires the C++ developer to call `ProcessEvents()` periodically OR call `Run()` to start the uv event loop.
     */
-    bool Initialize();
+    NODE_EXTERN void Initialize(const std::string& program_name);

```

Comment:
Would like this parameter to be optional, since it was mainly used to set the process name of the node application.